### ISSUE TITLE
Debian build script broken

### ISSUE THREAD

**Issue by terion-name** [github/git-lfs#985]
It seems the same as #388, but it till doen't work.
Ubuntu 14.04
```
root@ubuntu-1gb-ams2-01:~# export GOPATH=$PWD/go
root@ubuntu-1gb-ams2-01:~# mkdir -p go/src/github.com/github
root@ubuntu-1gb-ams2-01:~# cd go/src/github.com/github
root@ubuntu-1gb-ams2-01:~/go/src/github.com/github# git clone https://github.com/github/git-lfs.git
Cloning into 'git-lfs'...
remote: Counting objects: 13277, done.
remote: Compressing objects: 100% (34/34), done.
remote: Total 13277 (delta 8), reused 0 (delta 0), pack-reused 13243
Receiving objects: 100% (13277/13277), 4.63 MiB | 2.51 MiB/s, done.
Resolving deltas: 100% (8839/8839), done.
Checking connectivity... done.
root@ubuntu-1gb-ams2-01:~/go/src/github.com/github# cd git-lfs/
root@ubuntu-1gb-ams2-01:~/go/src/github.com/github/git-lfs# script/debian-build
```

```
....
+ cd /root/go/src/github.com/github/git-lfs
+ ./script/bootstrap
can't load package: package github.com/github/git-lfs/commands: cannot find package "github.com/github/git-lfs/commands" in any of:
	/root/.gvm/gos/go1.4.2/src/github.com/github/git-lfs/commands (from $GOROOT)
	/root/.gvm/pkgsets/go1.4.2/global/src/github.com/github/git-lfs/commands (from $GOPATH)
can't load package: package github.com/github/git-lfs/lfs: cannot find package "github.com/github/git-lfs/lfs" in any of:
	/root/.gvm/gos/go1.4.2/src/github.com/github/git-lfs/lfs (from $GOROOT)
	/root/.gvm/pkgsets/go1.4.2/global/src/github.com/github/git-lfs/lfs (from $GOPATH)
script/build.go:15:2: cannot find package "github.com/github/git-lfs/lfs" in any of:
	/root/.gvm/gos/go1.4.2/src/github.com/github/git-lfs/lfs (from $GOROOT)
	/root/.gvm/pkgsets/go1.4.2/global/src/github.com/github/git-lfs/lfs (from $GOPATH)
++ echo FAIL
FAIL
```

**Comment by terion-name** [comment]
BTW, after that I've run:
```
root@ubuntu-1gb-ams2-01:~# wget https://github.com/github/git-lfs/releases/download/v1.1.1/git-lfs-linux-amd64-1.1.1.tar.gz
root@ubuntu-1gb-ams2-01:~# tar -xvzf git-lfs-linux-amd64-1.1.1.tar.gz
git-lfs-1.1.1/
git-lfs-1.1.1/CHANGELOG.md
git-lfs-1.1.1/git-lfs
git-lfs-1.1.1/install.sh
git-lfs-1.1.1/README.md
root@ubuntu-1gb-ams2-01:~# cd git-lfs-1.1.1/
root@ubuntu-1gb-ams2-01:~/git-lfs-1.1.1# ls
CHANGELOG.md  README.md  git-lfs  install.sh
root@ubuntu-1gb-ams2-01:~/git-lfs-1.1.1# ./install.sh
```
And immediately got:
```
Git LFS initialized.
```

So it seems that build script installs LFS successfully, but exits with error?

**Comment by technoweenie** [comment]
Ah, it's because the script is installing gvm if go isn't installed. The stack trace only shows `/root/.gvm/pkgsets/go1.4.2/global` in your `GOPATH`, not `/root/go`. That script isn't up to date right now. 

@andyneff or @ssgelm: Think it's worth keeping these older install scripts up to date?

We have  [deb files](https://packagecloud.io/github/git-lfs?filter=debs) linux installs, but they didn't upload for Ubuntu for some reason. I'll fix that today.

**Comment by technoweenie** [comment]
@terion-name: In your second attempt, you downloaded a statically compiled `git-lfs` binary. There's no need to run `script/debian-build` script for it.

**Comment by ssgelm** [comment]
My vote would be to nuke the script and instead have people use the debs (which they can build themselves if they so choose) but I have a clear bias towards packages :)

**Comment by andyneff** [comment]
I've actually thought about this many times, and never bothered to mention it. I agree with @ssgelm, nuke em.

The scripts were once important before the packages because of how hard it is to get the ruby ronn for the man pages... This isn't important except for the packages. So they served their purpose, but now if someone want to help develop, all they need is go and the knowledge to call the bootstrap script, and I'd argue a go dev can manage that on their own.

**Comment by technoweenie** [comment]
Cool, I'll drop both the debian and rpm scripts :+1:

**Comment by andyneff** [comment]
Arg. We'll have to patch this first

https://github.com/andyneff/git-lfs_dockers/blob/master/centos_7.bootstrap#L33

**Comment by technoweenie** [comment]
Yes, these scripts: #1026

**Comment by javabrett** [comment]
I think this is resolved by #1026?

**Comment by ttaylorr** [comment]
Yep, closing for now 😄
